pkg/build/scanner.go

```diff
@@ -73,8 +73,8 @@ func Scanner(manifest model.Manifest, githubToken, git, branch string) error {
 	// Time format chosen for consistency with build tools tag:
 	// https://github.com/istio/tools/blob/ee7da00900dc878a2e865e43250c34735f130b7a/docker/build-tools/build-and-push.sh#L27
 	const timeFormat = "2006-01-02T15-04-05"
-	buildTimestamp := time.Now().Format(timeFormat)
-	log.Infof("new base tag: %s", buildTimestamp)
+	tag := fmt.Sprintf("%s-%s", manifest.Version, time.Now().Format(timeFormat))
```

Comment:
nit: the build-tools uses the actual branch, not version. Version is slightly different.

I am not against this, but if we already have to figure out the branch for `:latest` I slightly prefer using thebranch for consistency